src/middlewared/middlewared/plugins/iscsi_/target_to_extent.py

```diff
@@ -1,7 +1,9 @@
-import middlewared.sqlalchemy as sa
+import asyncio
 
-from middlewared.schema import accepts, Bool, Dict, Int, Patch
-from middlewared.service import CallError, CRUDService, private, ValidationErrors
+import middlewared.sqlalchemy as sa
+from middlewared.schema import Bool, Dict, Int, Patch, accepts
+from middlewared.service import (CallError, CRUDService, ValidationErrors,
```

Comment:
Can you fix the import? I'm not sure if it's your IDE, but our flake8 checks are 120 characters on a single line which also includes imports. No reason to wrap this.